### DISCUSSION TITLE
How to download a full resolution image from easyzoom.com

### DISCUSSION THREAD

**Discussion by fanblade** [lovasoa/dezoomify-rs#220] [Show and tell]
dezoomify-rs can be used to download a full resolution image from easyzoom.com when the author has disabled the download button. Here are the steps that worked for me on Windows Chrome:
1. In the browser window that contains your zoomable image, open Chrome's Developer Tools pane (press F12).
2. Under the Network tab, select Fetch/XHR.
3. Look for a 1-2kB XHR file with a string of 32 random characters for its name. Select it.
4. Near the bottom of the file's content you'll find a node named TilesZipUrl. Its value is a URL for an szi file. Copy this URL and paste it into a new browser tab.
5. The tab will take a while to download the szi file, then will display it as an icon on a black background. Right-click the icon and select "Save As..." to save it to disk.
6. Once downloaded to disk, rename the file to end with .zip.
7. Extract the zip file to disk. Find a dzi file within the extracted location.
8. Download the latest version of dezoomify-rs.exe here: https://github.com/lovasoa/dezoomify-rs/releases
9. Execute dezoomify-rs.exe from the Windows command line.
10. Paste the path to the dzi file extracted in step 7. For example, C:\0b29f1a2-41fa-4f60-9e8b-f435405dee88\0b29f1a2-41fa-4f60-9e8b-f435405dee88.dzi
11. Enter the zoom level you wish to "dezoomify" (0 for the full resolution).
12. A file named dezoomified.jpg will be saved to disk that contains the full resolution image.

**Comment by Yo55arian** [comment]
Does anyone know if this approach still works? I can find the szi URL but am unable to download it in Firefox or Chrome.

**Reply by StoiaCode** [reply]
Use JDownloader to download the file, they act like its a .jpg and the browser tries to display instead of download. Jdownloader will simply download no matter what.

**Reply by ackvf** [reply]
Yes, it still works. I see thousands of szi requests in Network tab, just pick one and copy the url
<img width="1545" height="1990" alt="image" src="https://github.com/user-attachments/assets/37ffe1d8-0aa0-41d3-bdbb-4fd931e02185" />

Remove the query string
```
https://easyzoom.blob.core.windows.net/tiled/e1829649-09db-48a5-875b-8913d7d773c1/e1829649-09db-48a5-875b-8913d7d773c1.szi?bytes=18856444-18865719

https://easyzoom.blob.core.windows.net/tiled/e1829649-09db-48a5-875b-8913d7d773c1/e1829649-09db-48a5-875b-8913d7d773c1.szi
```
Open that in the browser and then download the page with Ctrl+S or right click.

<img width="1119" height="766" alt="image" src="https://github.com/user-attachments/assets/102353ce-3c1a-47da-91e8-7b72334968a9" />

After download rename the file to .zip and extract.

`e1829649-09db-48a5-875b-8913d7d773c1.szi` -> `e1829649-09db-48a5-875b-8913d7d773c1.szi.zip`

You will see several folders with progressively increasing level of detail.

<img width="1419" height="935" alt="image" src="https://github.com/user-attachments/assets/3f361036-d503-4749-81a2-7c5d6247c13b" />

Next download the executable and place it alongside the `.dzi` file for convenience. Execute it and paste the name of it without any path segment (as it sits next to it).

<img width="858" height="264" alt="image" src="https://github.com/user-attachments/assets/9f34ea73-6a52-4794-a57c-59597622c7b5" />

Choose zoom level (write the number), hit Enter and that's it:

<img width="564" height="355" alt="image" src="https://github.com/user-attachments/assets/2502982c-a6f0-4a7e-8aea-5c6e445001d6" />

Your file will appear in the same folder.

```
> dir
06.02.2026  03:53        21 898 468 dezoomified.jpg  <- 22 MB
06.02.2026  03:48        15 127 552 dezoomify-rs.exe
07.05.2022  07:52               206 e1829649-09db-48a5-875b-8913d7d773c1.dzi
06.02.2026  03:37    <DIR>          e1829649-09db-48a5-875b-8913d7d773c1_files
```
